test/org/sagebionetworks/bridge/models/surveys/TestSurvey.java

```diff
@@ -162,7 +163,7 @@ public static SurveyQuestion selectBy(Survey survey, DataType type) {
     public TestSurvey(boolean makeNew) {
         setGuid(UUID.randomUUID().toString());
         setName("General Blood Pressure Survey");
-        setIdentifier("bloodpressure");
```

Comment:
So, if the user deletes a survey with identifier A, and later adds one back under A, what happens? Is it broken?